WiseUnpacker/SabreTools.Models.WiseInstaller/SectionHeader.cs

```diff
@@ -136,10 +136,44 @@ public class SectionHeader
         /// Byte array representing version. Byte array used due to unknown size and type for version.
         /// </summary>
         public byte[]? Version { get; set; }
+        
+        /// <summary>
+        /// String representing the WIS[etc].TMP string
+        /// </summary>
+        public string? TmpString { get; set; }
+        
+        /// <summary>
+        /// String representing the GUID string.
+        /// </summary>
+        public string? GuidString { get; set; }
+        
+        /// <summary>
+        /// String representing a version number. This isn't the version of the .WISE installer itself, as it is
+        /// entirely inconsistent even within the same week. Likely refers to a version for what's being installed
+        /// rather than the installer itself
+        /// </summary>
+        public string? NonWiseVersion { get; set; }
+
+        /// <summary>
+        /// Unknown. May also refer to a non-value for pre-78-offset executables and only a value for 78-offset-onwards
+        /// ones.
+        /// </summary>
+        public byte[]? PreFontValue {get; set;}
+        
+        /// <summary>
+        /// Font size
+        /// </summary>
+        public int FontSize { get; set; }
+        
+        /// <summary>
+        /// Byte array representing string lengths and info. Individual strings not predefined since number of strings
+        /// will likely vary between many installers.
+        /// </summary>
+        public byte[]? StringValues { get; set; }
```

Comment:
Wait, is there a difference between this and `Strings` immediately below? Or are these duplicate values?

Reply:
StringValues is the byte array that holds the pre-string byte values that tell you how to read the following strings